config/initializers/reserved_words.rb

```diff
@@ -190,7 +190,6 @@ class ReservedWords
     start
     started
     startups
-    swagnets
```

Comment:
I would leave this in though.

I searched for the definition and it looks like we shouldn't allow people to register that as a username: https://www.urbandictionary.com/define.php?term=Swagnet :-|

Reply:
Ok, ket's keep it, I was in doubt about it.